docs/contribute_to_linode_using_hugo.md

```diff
@@ -0,0 +1,104 @@
+
+
+## Clone the Docs Repo
+
+All of our guides are stored in the [github.com/linode/docs](https://github.com/linode/docs) repository. You will need to clone this repository to your local computer:
+
+1.  Install Git:
+
+        sudo apt install git
+        git config --global user.email "your.email@example.com"
+        git config --global user.name  "Your Name"
+
+2.  Clone `github.com/linode/docs`:
+
+        git clone https://github.com/linode/docs
+
+3.  Navigate to the new directory:
+
+        cd docs
+
+## Install Hugo
+
+The Linode documentation site is built using [Hugo](http://gohugo.io), a static site generator. In order to preview your guide before submission, you will need to install Hugo on your local computer.
+
+
+1.  On OSX, the easiest way to install Hugo is by using [Homebrew](https://brew.sh/):
+
+        brew install hugo
+
+2.  For other platforms, go to the [Hugo releases](https://github.com/gohugoio/hugo/releases) page and download the most up to date binary for your platform.
+
+        curl -OL https://github.com/gohugoio/hugo/releases/download/v0.30.2/hugo_0.30.2_Linux-64bit.tar.gz
+        tar -xvzf hugo_0.30.2_Linux-64bit.tar.gz
+        sudo mv hugo /usr/local/bin
+
+{{< caution >}}
+We recommend using Hugo version >=0.30. Earlier versions will not render our documentation correctly.
+{{< /caution >}}
+
+## Create a New Guide
+
+This section will take you through the process of creating a new guide using the topic of installing nginx on Debian as an example. You can use a Hugo [archetype](https://gohugo.io/content-management/archetypes/) to simplify the process
+
+1. Create a new branch for your guide:
+
+        git checkout -b nginx-on-debian
+
+2. From the root of the `docs` repo, run the following command. Specify the location and title of your guide; the example nginx guide should be located in `web-servers/nginx`:
+
+        hugo new web-servers/nginx/how-to-install-nginx-on-debian.md --kind content
+
+3. Start the Hugo server:
+
+        hugo server
+
+4. In a web browser, navigate to the location of your new guide. The example nginx guide will be located at `http://localhost:1313/docs/web-servers/nginx/how-to-install-nginx-on-debian`.
+
+## Write and Submit
+
+Your local Hugo development server has hot-reloading enabled, so you will be able to view changes to your guide as you write them. Please see our [Linode Writer's Formatting guide](/docs/linode-writers-formatting-guide/) and our [Linode Style guide](/docs/doesnt/exist/yet) for more information.
+
+{{< note >}}
+If your guide requires any images, these will go in the `content/assets` folder. Create a new directory for your guide and place all of the images there.
```

Comment:
Are we enforcing creating a new folder with the title of the guide to store static assets ?